plugins/alerts/api/alertModules/rating.js

```diff
@@ -0,0 +1,186 @@
+const log = require('../../../../api/utils/log.js')('alert:metric');
+
+const RatingAlert = {
+
+    /**
+	 * function for sending alert email
+	 * @param {object} alertConfigs  - alertConfig record from db
+	 * @param {object} result - alert data for email template
+	 * @param {function} callback - callback after calling email
+     * @return {object} promise object
+	 */
+    alert(alertConfigs, result, callback) {
+
+        return bluebird.coroutine(function *() {
+            try {
+                log.i('trigger alert:', alertConfigs);
+                utils.addAlertCount();
+                if (alertConfigs.alertBy === 'email') {
+                    const emails = yield utils.getDashboardUserEmail(alertConfigs.alertValues); //alertConfigs.alertValues.split(',');
+                    let html = '';
+                    const host = yield utils.getHost();
+
+                    let appsListTitle = 'several apps';
+                    if (result.length <= 3) {
+                        const appName = [];
+                        result.map((data)=>{
+                            appName.push(data.app.name);
+                        });
+                        appsListTitle = appName.join(', ');
+                    }
+
+                    let title = '';
+                    let keyName = alertConfigs.alertDataSubType;
+                    title = `${keyName} count for ${appsListTitle} has changed compared to yesterday`;
+                    const subject = title;
+
+                    html = yield utils.getEmailTemplate({
+                        title: `Countly Alert`,
+                        subTitle: `Countly Alert: ` + alertConfigs.alertName,
+                        host,
+                        compareDescribe: alertConfigs.compareDescribe,
+                        apps: result.map((data)=>{
+                            const item = {
+                                id: data.app._id,
+                                name: data.app.name,
+                                data: [{
+                                    key: 'Today\'s Value',
+                                    value: data.todayValue
+                                }]
+                            };
+                            if (data.lastDateValue !== null && data.lastDateValue !== undefined) {
+                                item.data.push({key: 'Yesterday\'s Value', value: data.lastDateValue});
+                            }
+                            return item;
+                        })
+                    });
+                    emails.forEach((to) => {
+                        utils.addAlertCount(to);
+                        log.i('will send email=>>>>>>>>>>');
+                        log.i('to:', to);
+                        log.d('subject:', subject);
+                        log.d('message:', html);
+                        utils.sendEmail(to, subject, html);
+                    });
+                    callback && callback();
```

Comment:
Codacy found an issue: [Expected an assignment or function call and instead saw an expression.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=6322208)